circle.yml

```diff
@@ -371,16 +372,14 @@ jobs:
   sanitizers-clang:
     executor: linux-clang-latest
     environment:
-      # TODO: Enable detect_stack_use_after_return=1 when https://bugs.llvm.org/show_bug.cgi?id=47626 is fixed.
-      ASAN_OPTIONS: detect_invalid_pointer_pairs=2
+      ASAN_OPTIONS: detect_stack_use_after_return=1:check_initialization_order=1
       UBSAN_OPTIONS: halt_on_error=1
     steps:
       - install_testfloat
       - checkout
       - build:
           build_type: RelWithDebInfo
-          # TODO: pointer-compare produces failure in std::string operator+.
```

Comment:
This didn't even use pointer-compare, but I see you removed pointer-subtract -- perhaps that should be removed in the first commit?

Reply:
Done.

Reply:
It seem it only removed the comment and not the line below?